### ISSUE TITLE
bug: check required to return "ok" property

### ISSUE THREAD

**Issue by lsloan** [mwarkentin/django-watchman#132]
I wrote the following custom check for django-watchman.  If it doesn't return an "ok" property, errors and a stack trace are shown in the output.

```python
@check
def jobsCheck() -> dict:
    jobIntervalMinutes: int = 15
    jobIntervalToleranceMinutes: int = 2
    jobIntervalSeconds: int = (jobIntervalMinutes + jobIntervalToleranceMinutes) * 60

    now: datetime = datetime.now(tzutc())
    latestMessage: JobLog = JobLog.objects.order_by('-' + JobLog.timestamp.field_name).first()

    latestIntervalSeconds: int = (now - latestMessage.timestamp).total_seconds()

    return {
        'jobs': {
            'latestTimestamp': latestMessage.timestamp.isoformat(),
            'latestIntervalSeconds': latestIntervalSeconds,
            'latestIntervalOk': latestIntervalSeconds < jobIntervalSeconds,
            # 'ok': latestIntervalSeconds < jobIntervalSeconds,  # django-watchman requires "ok"; undocumented
        }
    }
```

When run with the "ok" property commented out, as shown above, I get the following response:

```json
{"error": 500, "message": "'ok'", "traceback": "Traceback (most recent call last):\n  File \"/Users/lsloan/anaconda3/envs/myapp-dev/lib/python3.6/site-packages/jsonview/decorators.py\", line 74, in _wrapped\n    ret = f(request, *a, **kw)\n  File \"/Users/lsloan/anaconda3/envs/myapp-dev/lib/python3.6/site-packages/watchman/views.py\", line 81, in status\n    checks, ok = run_checks(request)\n  File \"/Users/lsloan/anaconda3/envs/myapp-dev/lib/python3.6/site-packages/watchman/views.py\", line 65, in run_checks\n    if not result['ok']:\nKeyError: 'ok'\n"}
```

If I uncomment the "ok" property, all works well:

```json
{"storage": {"ok": true}, "databases": [{"default": {"ok": true}}], "jobs": {"latestTimestamp": "2018-12-03T19:27:27.240002+00:00", "latestIntervalSeconds": 1918.825879, "latestIntervalOk": false, "ok": false}}
```

I'm using macOS 10.13.2 and for Python, "Python 3.6.5 :: Anaconda, Inc."

**Comment by mwarkentin** [comment]
Thanks for the report - is the gist that you'd like to be able to specify your own field to be the "ok" check (in this case `latestIntervalOk`)? What do you see as the benefit of this over just renaming your field to `ok`? 

I don't see a technical reason this wouldn't work, though it would make things a bit more complex so I'd like to make sure there's an enticing use case. We would need some way for a check to specify the name of the "ok" field for watchman (perhaps it defaults to `ok` but can be overridden).

Alternatively, would improving the documentation around this part of custom checks be sufficient?

Thanks!

**Comment by lsloan** [comment]
Yes, improving the documentation (and examples and docstrings) would be a big help.  I didn't see any mention in the documentation that `ok` must be in the results.  It took me some time to figure out why my check was failing.  It would help if the docstring for `@check` stated that `ok` is required.

In my example, I like having the property `latestIntervalOk` in the results because it's a kind of self-documentation of what's being checked.  I've changed my code since I reported this issue (it was too simple, needed to handle exceptions, etc.), so it now stores the result in a variable and the results are returned with:

```python
    return {
        'jobs': {
            'latestTimestamp': latestTimestamp,
            'latestIntervalSeconds': latestIntervalSeconds,
            'latestIntervalOk': latestIntervalOk,
            'ok': latestIntervalOk,  # watchman requires "ok"; undocumented
        }
    }
```

I may use the dashboard feature in the near future, too. I see that the `ok` property is required to make that function properly.  I think it's not necessary to support an alternate for `ok`.  That'll probably make things too complicated.

That's my long-winded response to say updated documentation is good enough.
